prisma/seed.ts

```diff
@@ -50,6 +76,22 @@ const groups = [
 
 async function seed() {
   try {
+    for (const user of users) {
+      await db.user.create({
```

Comment:
Not a blocker since this is not production code but a one-off seeding script but maybe we can use `createMany` to avoid 10 separate requests to the database.